R/valid.R

```diff
@@ -44,6 +117,8 @@ valid_eventually <- function(object, fun) {
   out <- fun(object)
   attr(out, ".should_validate") <- old
   validate(out)
+
+  out
```

Comment:
Were you trying to change the visibility? Should you also do this in `valid_implicitly()`?